posthog/models/group.py

```diff
@@ -0,0 +1,27 @@
+from django.db import models
+
+
+class Group(models.Model):
```

Comment:
There's already a model called group in cohort groups 🙈

Reply:
That's not a model, just an internal helper. But we can always rename it :)